rpc/client.go

```diff
@@ -131,6 +133,15 @@ func NewClient(addr net.Addr, context *Context) *Client {
 		disableReconnects: context.DisableReconnects,
 		clock:             context.localClock,
 		remoteClocks:      context.RemoteClocks,
+		heartbeatInterval: context.heartbeatInterval,
+		heartbeatTimeout:  context.heartbeatTimeout,
+	}
+
+	if c.heartbeatInterval == 0 {
+		c.heartbeatInterval = defaultHeartbeatInterval
+	}
+	if c.heartbeatTimeout == 0 {
+		c.heartbeatTimeout = defaultHeartbeatTimeout
```

Comment:
I think `c.heartbeatTimeout = 2 * c.heartbeatInterval` is still a better default than a hard-coded 6s (which basically forces anyone who sets one to set the other). The test should set an explicit `heartbeatTimeout` instead of relying on the default computed one.